### DISCUSSION TITLE
Dependabot alerts failing with 500

### DISCUSSION THREAD

**Discussion by vezzick** [community/community#52471] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

The github dependabot alerts API has been failing with 500s with this error message:

invalid json response body at https://api.github.com/repos/[REPO-NAME]/dependabot/alerts?per_page=100&type=all&state=open reason: Unexpected end of JSON input

this happened for a few hours on March 31st as well.

Nothing on https://www.githubstatus.com/ or in the incident history.

the api in question:
https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28

any way to find out what is going on here?

**Comment by jdstrand** [comment]
I'm also seeing this with `https://api.github.com/orgs/[ORG]/dependabot/alerts`, but the response body is empty:

```
$ curl -i -L -H "Authorization: Bearer $GHTOKEN" -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/orgs/[ORG]/dependabot/alerts
HTTP/2 500 
server: GitHub.com
date: Mon, 10 Apr 2023 20:12:52 GMT
content-type: application/json; charset=utf-8
content-length: 0
x-oauth-scopes: read:org, repo
x-accepted-oauth-scopes: public_repo, repo, security_events
github-authentication-token-expiration: 2023-04-28 20:15:56 UTC
x-github-media-type: github.v3; format=json
link: <https://api.github.com/organizations/REDACTED>; rel="next"
x-github-api-version-selected: 2022-11-28
x-ratelimit-limit: 5000
x-ratelimit-remaining: 4989
x-ratelimit-reset: 1681159159
x-ratelimit-used: 11
x-ratelimit-resource: core
access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
access-control-allow-origin: *
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-frame-options: deny
x-content-type-options: nosniff
x-xss-protection: 0
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
content-security-policy: default-src 'none'
vary: Accept-Encoding, Accept, X-Requested-With
x-github-request-id: C452:0B66:244CE:4B48A:64346DC3

```

secret-scanning and code-scanning are functioning normally.

**Reply by jdstrand** [reply]
UPDATE: `https://api.github.com/repos/[ORG]/[REPO]/dependabot/alerts` appears to be working now, but `https://api.github.com/orgs/[ORG]/dependabot/alerts` still is not.

**Reply by vezzick** [reply]
im using the npm package so it's probably the same empty result but mine is wrapped in some fetch library's error output

**Comment by jdstrand** [comment]
FYI, this is working again. I know it worked ~10 hours ago.
